transforms/code/header_cleanser/ray/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM docker.io/rayproject/ray:2.24.0-py310
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Install system dependencies, including libgomp1
+RUN sudo apt-get update && sudo apt-get install -y \
```

Comment:
can you move this down below install of python_transform. it will give better  image caching during ci/cd.